docs/using/config/usage.rst

```diff
@@ -44,10 +44,11 @@ To authenticate with the control service, you will need a copy of the public clu
 
 For ``flocker-deploy``, your API user certificate and key should be in files named ``user.crt`` and ``user.key`` and the cluster certificate in file ``cluster.crt``.
 
-By default, ``flocker-deploy`` will look for these certificate files in the current working directory.
-If this is not where the files are located, you may specify the ``--certificates-directory`` option to ``flocker-deploy``:
+By default, ``flocker-deploy`` will look for these certificate files in the current working directory and expect them to be named ``cluster.crt`` (the public cluster certificate), ``user.crt`` (the API user certificate) and ``user.key`` (the API user's private key).
+
+You can override any of these defaults with the ``--cafile`` (cluster certificate), ``--cert`` (user certificate) and ``--key`` (user private key) options, specifying the full path to each file.
```

Comment:
As per normal Linux behaviour (don't know about OS X but I'd imagine it's the same), if the files are in your working directory, you can just specify the file name without an absolute path. Perhaps changing to "specifying the path to each file" would suffice.....I'm not sure. But generally in Linux, the "full path" rather than the "absolute path" is technically just the file name if it's in your CWD. Anyway, if you can think of a good wording suggestion for this sentence, please add it in a comment :)